src/controllers/expensesController.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const userService = require('../services/users');
```

Comment:
Use destructuring to import multiple exports from the same module: const { getUser } = require('../services/users');